docs/ocs/troubleshooting.md

```diff
@@ -0,0 +1,169 @@
+# Troubleshooting OpenShift Container Storage (OCS)
```

Comment:
Just to be consistent with out other trouble shooting docs, can we name this: 

`# OpenShift Container Storage (OCS) Runbook` 

and for the filename instead of `troubleshooting.md` name it `runbook.md` ?

Reply:
That seems odd to me; it's not really a runbook; it's *just* troubleshooting. I would expect a runbook to be a much more general document.

Reply:
@larsks -- then I'm not sure I understand the difference between troubleshooting and the purpose a runbook. Isn't the point of a runbook in large part to help with troubleshooting?

Reply:
I don't know, I always thought a run book was "how to operate this thing", of which troubleshooting is only a part. I guess I consider all the documents in this directory to constitute parts of a run book for OCS.